subsys/suit/stream/stream_filters/include/suit_decompress_filter.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SUIT_DECOMPRESS_FILTER_H__
+#define SUIT_DECOMPRESS_FILTER_H__
+
+#include <suit_sink.h>
+#include <suit_types.h>
+#include <suit_metadata.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Interface function type for output sink readback, passed to
+ * suit_decompress_filter_get() as a callback. Decompression process needs
+ * an access to unaltered destination component memory (see WARNING below).
+ *
+ * @param[in] sink_ctx Context of decompression output sink
+ * @param[in] offset Offset of output sink area to start reading from
+ * @param[out] buf Buffer to read into
+ * @param[in] size Size of @a buf; data read size
+ */
+typedef suit_plat_err_t (*out_sink_readback_func)(void *sink_ctx,
+							size_t offset, uint8_t *buf, size_t size);
```

Comment:
alignment is off here and below, and line length is 100, sink_ctx, offset and buf can al be on the top line and is under limit